test/EmailAddressTest.php

```diff
@@ -232,6 +232,13 @@ public function validEmailAddresses()
             'bob@verylongdomainsupercalifragilisticexpialidociousspoonfulofsugar.com' => ['bob@verylongdomainsupercalifragilisticexpialidociousspoonfulofsugar.com'],
             "B.O'Callaghan@domain.com"                                                => ["B.O'Callaghan@domain.com"],
         ];
+        
+        if (extension_loaded('intl')) {
+            $return['bob@موقع.إختبار']             = ['bob@موقع.إختبار'];
+            $return['bob@xn--4gbrim.xn--kgbechtv'] = ['bob@xn--4gbrim.xn--kgbechtv'];
```

Comment:
I've merged #67 at this point, and just attempted to merge this PR. When running tests, both of these specific cases fail, each with:

```text
not a valid hostname for the email address
The input appears to be a DNS hostname but cannot match TLD against known list
The input appears to be a local network name but local network names are not allowed
Failed asserting that false is true.
```

I'm not sure if that's _expected_ at this point, and you have more work to do, or if there's another, more subtle issue.